lib/matplotlib/tests/test_axes.py

```diff
@@ -2542,6 +2542,18 @@ def test_hist_stacked_normed():
     ax.hist((d1, d2), stacked=True, normed=True)
 
 
+@image_comparison(baseline_images=['hist_stacked_normed'])
```

Comment:
If you are going to add an image test you also need to commit the baseline images.

If this adds coverage we don't have other wise the images (or at least the png) should be added, if not, just drop the decorator.